packages/ui/src/components/SnackBar/SnackStackProvider.tsx

```diff
@@ -0,0 +1,67 @@
+import { useState, useContext, createContext } from "react";
+import { ToastType } from "../ToastContent";
+import Snackbar from "./Snackbar";
+
+export type ToastProps = {
+  key?: number;
+  title?: string;
+  message?: string;
+  link?: string;
+  duration?: number;
+  type: ToastType;
+  position?: {
+    vertical?: "top" | "bottom";
+    horizontal?: "left" | "right" | "center";
+  };
```

Comment:
you can use the type `SnackbarOrigin` right away
```suggestion
  position?: SnackbarOrigin;
```